src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -251,14 +254,48 @@
       [:div {}
        [comps/Button
         {:text "Sign In"
-         :href (str (config/api-url-root) "/login?callback="
-                    (js/encodeURIComponent
-                     (str js/window.location.protocol "//" js/window.location.hostname))
-                    (let [hash (nav/get-hash-value)]
-                      (if (clojure.string/blank? hash) "" (str "&path=" hash))))}]]
+         :onClick (fn [e]
+                    (.. js/window
+                        (open
+                         (str (config/api-url-root) "/login?callback="
+                              (js/encodeURIComponent (.. js/window -location -origin)))
+                         "Authentication"
+                         "menubar=no,toolbar=no,width=500,height=500")))}]]
       [:div {:style {:marginTop "1em"}} [RegisterLink]]
       [:div {:style {:maxWidth 600 :paddingTop "2em" :fontSize "small"}}
-        [Policy]]])})
+        [Policy]]])
+   :component-did-mount-post-message
```

Comment:
delete component-did-mount-post-message and component-will-unmount-post-message? Those aren't used now, right?